normandy/settings.py

```diff
@@ -399,7 +399,7 @@ class Development(Base):
 class Production(Base):
     """Settings for the production environment."""
 
-    USE_X_FORWARDED_HOST = values.BooleanValue(True)
+    USE_X_FORWARDED_HOST = values.BooleanValue(False)
```

Comment:
`False` is the default in the `Base` config. Lets just delete the line instead of duplicating it here.